pyro/contrib/timeseries/gp.py

```diff
@@ -0,0 +1,144 @@
+import torch
+import torch.nn as nn
+from torch.distributions import MultivariateNormal
+
+import pyro.distributions as dist
+from pyro.ops.ssm_gp import MaternKernel
+
+
+class TimeSeriesModel(nn.Module):
```

Comment:
Can you move this to a separate file, say named `base.py`?

Reply:
ok